packages/core/src/web/app/components/dialogs/DocumentSettings/index.tsx

```diff
@@ -476,26 +463,14 @@ const DocumentSettings = ({ unmount }: Props): React.JSX.Element => {
                   id="rotary_mode"
                   onChange={setRotaryMode}
                 />
+                <SettingFilled
+                  onClick={() =>
+                    showRotarySettings({ rotaryMode, workarea }, () => {
+                      setRotaryMode(BeamboxPreference.read('rotary_mode'));
+                    })
+                  }
+                />
                 {isCurveEngraving && renderWarningIcon(tGlobal.mode_conflict)}
-                {(addOnInfo.rotary.mirror || addOnInfo.rotary.extendWorkarea) && rotaryMode && (
-                  <>
-                    <div className={styles.subCheckbox}>
-                      {addOnInfo.rotary.extendWorkarea && (
-                        <Checkbox
-                          checked={extendRotaryWorkarea}
-                          onChange={(e) => setExtendRotaryWorkarea(e.target.checked)}
-                        >
-                          {tDocu.extend_workarea}
-                        </Checkbox>
-                      )}
-                      {addOnInfo.rotary.mirror && (
-                        <Checkbox checked={mirrorRotary} onChange={(e) => setMirrorRotary(e.target.checked)}>
-                          {tDocu.mirror}
```

Comment:
optional: move `extend_workarea` and `mirror` to `t.rotary_settings` after removed from document settings